bin/snapshot-backups/snapshot-purge

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+
+if [ -f $SCRIPT_DIR/../deploy/includes ]; then
+  . $SCRIPT_DIR/../deploy/includes
+else
+   echo Cannot find $SCRIPT_DIR/../deploy/includes
+   exit 1
+fi
+
+export BUCKET_SOURCE_FOLDER=public
+if [ -f ./bin/snapshot-backups/snapshot-include ]; then
+  . ./bin/snapshot-backups/snapshot-include
+else
+   echo Cannot find ${PWD}/bin/snapshot-backups/snapshot-include
+   exit 1
+fi
+if [ -f $SCRIPT_DIR/../deploy/includes ]; then
+  . $SCRIPT_DIR/../deploy/includes
+else
+   echo Cannot find $SCRIPT_DIR/../deploy/includes
+   exit 1
+fi
+
+# just testing?
+if [ x$1 = x"--dryrun" ]; then
+  echo=echo
+  dryrun=$1
+  shift
+fi
+
+SPACE=${1:-please-provide-space-name-as-first-argument}
+SPACE=$(echo "$SPACE" | tr '[:upper:]' '[:lower:]')
+assertCurSpace $SPACE
+shift
+
+removeTag=""
+removeTicket=""
+download=""
+
+while [ 0 -eq 0 ]; do
+
+  snaptype=$1
+
+  if [ x$snaptype = x ]; then
+    break
+  fi
+  shift
+
+  case "$snaptype" in
+
+      "--tag")
+        removeTag=$1
+        if [ x"$removeTag" == x ]; then
```

Comment:
Now I'm getting some usage guidance, but now I only see it for the first thing I provided (after $SPACE and possibly --dryrun), and only if they match one of the valid options. 

I would be in favor of adding a --help argument and having it just trigger a whole block of usage output, and of having anything else that looks squirrelly result in a message to use --help to see usage. Or something like that ...